client/api/src/main/java/org/eclipse/microprofile/graphql/client/GraphQLError.java

```diff
@@ -0,0 +1,35 @@
+package org.eclipse.microprofile.graphql.client;
```

Comment:
Can you add the copyright header?  It's probably easier to just add it, but it might make more sense to rename the previous file (Error.java) and then change the code (change class name and update javadoc) so that the history is preserved.  I don't think there is a lot of history there, so I'll leave it up to you. :)

Reply:
I'm no git expert, I attempted to persuade git that it's a rename rather than delete+create, but I wasn't successful (not when there are also textual changes in the file within the same commit)

Reply:
No worries - thanks for updating the copyright.